heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/RestClientWrapper.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.elasticsearch
+
+import com.spotify.heroic.common.Duration
+import com.spotify.heroic.elasticsearch.index.IndexMapping
+import eu.toolchain.async.AsyncFramework
+import org.apache.http.HttpHost
+import org.elasticsearch.client.RestClient
+import org.elasticsearch.client.RestHighLevelClient
+import org.elasticsearch.client.sniff.Sniffer
+import org.elasticsearch.search.aggregations.bucket.terms.ParsedStringTerms
+import java.net.InetAddress
+import java.net.UnknownHostException
+import java.util.concurrent.TimeUnit
+
+private const val DEFAULT_PORT = 9200
+
+data class RestClientWrapper @JvmOverloads constructor(
+    val seeds: List<String> = listOf("localhost"),
+    val sniff: Boolean = false,
+    val sniffInterval: Duration = Duration.of(5, TimeUnit.MINUTES)
```

Comment:
this is a high default -- iirc the default in the transport client was 30 seconds. 

make this at least 1 minute?

Reply:
Oops missed this before merging. This is the same default as the transport library, but yeah we set the default in Heroic to 30s. I'll make a new PR to update it.